src/minio/managers/policy_manager.py

```diff
@@ -181,6 +181,82 @@ async def get_group_policy(self, group_name: str) -> PolicyModel:
         """Retrieve the existing policy for a specific group."""
         return await self.get_policy(TargetType.GROUP, group_name)
 
+    # === POLICY ATTACHMENT OPERATIONS ===
+    async def attach_policy_to_user(self, policy_name: str, username: str) -> None:
```

Comment:
I'm wondering if a single method that has an enum for user / group defaulting to user and a boolean for attach / detach defaulting to detach might be easier to maintain than 4 methods, but this is fine